core/src/main/java/gov/nist/secauto/metaschema/core/metapath/function/library/DefaultFunctionLibrary.java

```diff
@@ -152,10 +152,13 @@ public DefaultFunctionLibrary() { // NOPMD - intentional
     // https://www.w3.org/TR/xpath-functions-31/#func-string
     registerFunction(FnString.SIGNATURE_NO_ARG);
     registerFunction(FnString.SIGNATURE_ONE_ARG);
+    // P0: https://www.w3.org/TR/xpath-functions-31/#func-string
+    registerFunction(FnSubstring.SIGNATURE_TWO_ARG);
+    registerFunction(FnSubstring.SIGNATURE_THREE_ARG);
     // P1: https://www.w3.org/TR/xpath-functions-31/#func-string-join
     // P1: https://www.w3.org/TR/xpath-functions-31/#func-string-length
     // P1: https://www.w3.org/TR/xpath-functions-31/#func-subsequence
-    // P1: https://www.w3.org/TR/xpath-functions-31/#func-substring
+    // https://www.w3.org/TR/xpath-functions-31/#func-substring
```

Comment:
```suggestion
    // https://www.w3.org/TR/xpath-functions-31/#func-substring
    registerFunction(FnSubstring.SIGNATURE_TWO_ARG);
    registerFunction(FnSubstring.SIGNATURE_THREE_ARG);

```

Reply:
It seems we have relocated it in the interim.